edx/completion
# Move logic into the completion repo

**Description:** 
Move the new logic created on edx-platform into new completion repo. 

**JIRA:** 
EDUCATOR-2189

**Dependencies:** 
None

**Testing instructions:**
Make sure the integration tests on edx-platform runs and passes

1. On openedx instance, turn on the waffle flags for completion.enable_completion_tracking and completion.enable_visual_progress
2. complete any vertical blocks
3. Make sure the horizontal courseware ribbon on top shows green checkmarks

**Reviewers:**
- [] @iloveagent57 

**Merge checklist:**
- [*] Version bumped
- [*] Changelog record added
- [*] PR author is listed in AUTHORS

**Post merge:**
      finished.
